### ISSUE TITLE
Path to flow cell run directory incorrect in DemultiplexingAPI

### ISSUE THREAD

**Issue by seallard** [Clinical-Genomics/cg#2316]
## Description
It seems like the path to the flow cell run directory is incorrectly set for non novaseq flow cells.

For example, when using the new demux post processing flow for a hiseq-x flow cell the following error is thrown:
```
Flow cell 170613_ST-E00266_0216_BH2FYJCCXY will be skipped: Sample sheet /home/proj/stage/flowcells/novaseq/runs/170613_ST-E00266_0216_BH2FYJCCXY/SampleSheet.csv does not exist in flow cell run directory.
```

But the sample sheet does exist, it is in another location
```
/home/proj/stage/flowcells/hiseqx/170613_ST-E00266_0216_BH2FYJCCXY/SampleSheet.csv
```

## Suggested solution
Either fix the definition in the `DemultiplexingAPI` or just fix it in the new post processing flow.

## This can be closed when
A hiseq-x flow cell can be post processed with the new flow. For example
```
cg demultiplex finish temporary 170613_ST-E00266_0216_BH2FYJCCXY
```

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by henrikstranneheim** [comment]
@seallard @diitaz93 Will this be fixed by: https://github.com/Clinical-Genomics/cg/issues/2337

**Comment by diitaz93** [comment]
> @seallard @diitaz93 Will this be fixed by: #2337

No, but maybe it could have been fixed when changing to a flat directory structure, what do you think @ChrOertlin

**Comment by seallard** [comment]
This has been resolved since we fetch the sample sheet from housekeeper now
